src/components/Navbar/Navbar.jsx

```diff
@@ -52,7 +52,7 @@ const Navbar = ({ width }) => {
                 className={classes.space}
                 variant={width !== 'xs' ? 'h6' : 'body1'}
               >
-                COVID Trial Dash{' '}
+                COVID Trial Volunteer Dashboard{' '}
```

Comment:
This is tough because it's not mobile friendly... Let's discuss

Reply:
maybe we remove the title when it's mobile and just keep the logo (currently doing the opposite)

Reply:
we can have it like this, and on mobile only show either the logo or the text. We can discuss later, but i'm ok with this change for now.